mods/persistence/modules/shuttles/shuttle.dm

```diff
@@ -1,6 +1,6 @@
 /datum/shuttle/New(_name, var/obj/effect/shuttle_landmark/initial_location)
 	if(!SSpersistence.in_loaded_world)
-		return ..()
+		return ..(_name, initial_location)
```

Comment:
why dis? fyi it forwards all args to ..() by default.

Reply:
At what point in time there were additional arguments I added until I found out a cleaner way to do it. Might not be necessary then either but probably an artifact of that. I'll clean it up here or in a future PR.